### DISCUSSION TITLE
基于 React 实现的站点

### DISCUSSION THREAD

**Discussion by zjc17** [YunYouJun/cook#29] [分享展示]
前段时间看到在线文档的时候想着可以学一下前端实现一下，碰巧又看到了你的项目，做的非常好看～

我用 `react` `nextjs` 实现了一个类似的[站点](https://cook.zhangjc.tech)，[仓库地址](https://github.com/zjc17/cook.zhangjc.tech)，并在关于界面添加了当前仓库的链接。

**Comment by YunYouJun** [comment]
怎么不试试 emoji？

**Reply by zjc17** [reply]
加上了🙈

**Comment by zjc17** [comment]
我用SSG+Nginx的方案打包docker镜像，最终的镜像大小仅12MB

**Reply by YunYouJun** [reply]
此处应该指的是部署用的镜像而非开发使用？

说实话，我自己开发和部署都未使用 Docker，所以具体需求并不了解。
一般期望的会是哪种形式？

**Reply by zjc17** [reply]
@YunYouJun 一般来讲，docker镜像是用来分发提供给有 self-hosting 需求的用户使用的。

用docker开发的话一般不用这样搞，可以直接在自定义的镜像里拉代码构建。

**Comment by YunYouJun** [comment]
@henryclw Can you have a look? Maybe we should change the way we provide docker now?

**Reply by zjc17** [reply]
@YunYouJun I've submitted a PR #34  for image optimizing.
